network/load-balancer/quickstart.mdx

```diff
@@ -6,78 +6,86 @@ content:
-      * By default, each Load Balancer is created using [round-robin](/network/load-balancer/concepts/#balancing-rules) balancing rules.
-      * You can [add additional frontends and backends](/network/load-balancer/how-to/manage-frontends-and-backends/) to your Load Balancer after its creation.
+    * By default, each Load Balancer is created using [round-robin](/network/load-balancer/concepts/#balancing-rules) balancing rules.
+    * You can [add additional frontends and backends](/network/load-balancer/how-to/manage-frontends-and-backends/) to your Load Balancer after its creation.
     </Message>
 
 ## How to add frontends
 
+You can add more frontends to your Load Balancer after its creation. This allows you to configure different frontends listening on different ports, and to connect different frontends to different backends as required.
+
 1. Click **Load Balancers** in the **Network** section of the [Scaleway Console](https://console.scaleway.com) side menu. The Load Balancer dashboard displays. 
 2. Click the Load Balancer you want to add frontends to. The Load Balancer information page displays. 
 3. Click the **Frontends** tab.
-4. Click the green **+** button.
-5. Enter a **name**, **port** and [**SSL certificate(s)**](/network/load-balancer/how-to/create-self-signed-tls-certificate/#how-to-import-a-self-signed-certificate-with-scaleway-load-balancers) for your new frontend. Note that each frontend must listen on a different port.
-6. Choose between **using an existing backend** or **creating a new backend**.
-    - If using an existing backend, click **Use an existing Backend** and select the backend of your choice from the dropdown list.
-    - If creating a new backend, click **Create a new Backend** and enter a name, protocol, port proxy, health check type, server IP(s) and advanced settings as required.
+4. Click the green **+** button to launch the frontend creation wizard.
+    <Lightbox src="scaleway-create-frontend.webp" alt="" />
+5. Enter a **name** and **port** for your new frontend. Note that each frontend must listen on a different port.
+6. Select an [**SSL certificate(s)**](/network/load-balancer/how-to/add-certificate) for your frontend to use, if required. Note that you must already have [created or imported an SSL certificate for your Load Balancer](/network/load-balancer/how-to/add-certificate/) in order to select one here.
+6. Choose between **using an existing backend** or **creating a new backend**. Your frontend will direct traffic to the backend you choose here.
+    - If using an existing backend, click **Use an existing backend** and select the backend of your choice from the dropdown list.
+    - If creating a new backend, click **Create a new backend** and enter a name, protocol, port proxy, health check type, server IP(s) and advanced settings as required. See our [dedicated documentation](/network/load-balancer/reference-content/configuring-load-balancer/) for help configuring your backend
```

Comment:
> enter a name, protocol, port proxy, health check type, server IP(s) and advanced settings as required

enter a name, protocol, port, proxy protocol, health check type, server IP(s) and advanced settings as required

Proxy alone is confusing, we are selecting a proxy protocol option here.